src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -844,6 +848,25 @@ bool fuse_type_to_multiclass_nms(const std::shared_ptr<ov::Node>& node, const pr
     });
 }
 
+bool fuse_type_to_multinomial_v13(const std::shared_ptr<ov::Node>& node, const precisions_map& precisions) {
+    auto multinomial = ov::as_type_ptr<opset13::Multinomial>(node);
+    if (!multinomial) {
+        return false;
+    }
+
+    auto it_output_type = precisions.find(node->get_output_element_type(0));
+    if (it_output_type == precisions.end())
+        return false;
+
+    const auto& output_type = it_output_type->second;
+    if (output_type.is_integral_number() || output_type.is_real()) {
+        multinomial->set_convert_type(ov::element::i32);
```

Comment:
should it be always i32? does this mean the plugin doesn't support i64?

Reply:
as I can see output_type always have to be integral, do we actually need to check "output_type.is_real()"?

Reply:
Yes, CPU plugin does not support i64 data type. It should not cause any inaccuracy errors as the data stored are indices of values <= previous layer size, so it is currently unfeasible to train a network that big.

Reply:
> as I can see output_type always have to be integral, do we actually need to check "output_type.is_real()"? ![image](https://private-user-images.githubusercontent.com/21181928/288756680-af2ee947-b727-44c3-90a5-00aa6b086294.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIyODQ2OTUsIm5iZiI6MTcwMjI4NDM5NSwicGF0aCI6Ii8yMTE4MTkyOC8yODg3NTY2ODAtYWYyZWU5NDctYjcyNy00NGMzLTkwYTUtMDBhYTZiMDg2Mjk0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjExVDA4NDYzNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgxYWMyNjZkMGU5YTdlZWRlZGVkMmYzYTVhMzQ3Zjk4Njc5NjA1Y2QzMGFlNjgzZTgzOGVkYTY5Mzg2YTc0YzkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.1b34vsFuK-fuoI29wuCFhtr5912_DXqVGvLGW3guslU)

Good call, we don't